src/borg/testsuite/archiver.py

```diff
@@ -2565,8 +2562,9 @@ def test_recreate_timestamp(self):
                  'test', archive)
         info = self.cmd('info', archive).splitlines()
         dtime = datetime(1970, 1, 2) + local_timezone.utcoffset(None)
-        assert any([re.search(r'Time \(start\).+ %s' % dtime.strftime("%Y-%m-%d"), item) for item in info])
-        assert any([re.search(r'Time \(end\).+ %s' % dtime.strftime("%Y-%m-%d"), item) for item in info])
+        s_time = dtime.strftime("%Y-%m-%d")
+        assert any([re.search(r'Time \(start\).+ %s' % s_time , item) for item in info])
```

Comment:
pep8 checker is unhappy:

src/borg/testsuite/archiver.py:2566:62: E203 whitespace before ','

Reply:
fixed now.